File: nl.naturalis.nba.client.php/model/Synonym.php
<?php

namespace nl\naturalis\nba\client\php\model;

use nl\naturalis\nba\client\php\model\NBADomainObject;
use nl\naturalis\nba\client\php\model\ScientificName;

class Synonym extends NBADomainObject {

	private $scientificName;
	private $taxa;

	/**
	 *
	 * @return \nl\naturalis\nba\client\php\model\ScientificName
	 */
	public function getScientificName()
	{
		return $this->scientificName;
	}

	public function setScientificName(ScientificName $scientificName)
	{
		$this->scientificName = $scientificName;
	}

	public function getTaxa()
	{
		return $this->taxa;
	}

	public function setTaxa(array $taxa)
	{
		$this->taxa = taxa;
	}
}

File: nl.naturalis.nba.api/src/main/java/nl/naturalis/nba/api/model/Synonym.java
package nl.naturalis.nba.api.model;

import java.util.List;

public class Synonym implements INbaModelObject {

	private ScientificName scientificName;
	private List<Taxon> taxa;

	public ScientificName getScientificName()
	{
		return scientificName;
	}

	public void setScientificName(ScientificName scientificName)
	{
		this.scientificName = scientificName;
	}

	public List<Taxon> getTaxa()
	{
		return taxa;
	}

	public void setTaxa(List<Taxon> taxa)
	{
		this.taxa = taxa;
	}

}
